data/map planets.txt

```diff
@@ -4699,6 +4711,9 @@ planet "Var' Kayi"
 	outfitter "Wanderer Advanced"
 	"required reputation" 5
 	bribe 0
+	tribute 4500
+		threshold 12500
+		fleet "Wanderer Defense" 7
```

Comment:
```suggestion
		fleet "Wanderer Defense" 12
```
As the only other place with the Wanderer Advanced shipyard, I'd say this ought to have a larger fleet, closer to that of Vara K'chrai.